spec.md

```diff
@@ -320,32 +329,32 @@ For success responses, the following fields are valid.
 }
 </pre>
 
-### Polling Interval and Duration    
+### Polling Interval and Duration
 
-The frequency and maximum duration of polling may vary by platform client. If a platform has a max polling duration and this limit is reached, the platform will cease polling and the operation state will be considered `failed`.
+The frequency and maximum duration of polling MAY vary by platform client. If a platform has a max polling duration and this limit is reached, the platform will cease polling and the operation state will be considered `failed`.
```

Comment:
I think "will be considered `failed`" could use some consideration - I think 'will' here is probably equivalent to a MUST.